mod-source-record-storage-server/src/main/java/org/folio/services/util/AdditionalFieldsUtil.java

```diff
@@ -0,0 +1,92 @@
+package org.folio.services.util;
+
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.folio.rest.jaxrs.model.Record;
+import org.marc4j.MarcJsonReader;
+import org.marc4j.MarcJsonWriter;
+import org.marc4j.MarcReader;
+import org.marc4j.MarcStreamWriter;
+import org.marc4j.MarcWriter;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.MarcFactory;
+import org.marc4j.marc.VariableField;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+/**
+ * Util to work with additional fields
+ */
+public class AdditionalFieldsUtil {
+
+  public static final String TAG_999 = "999";
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(AdditionalFieldsUtil.class);
+  private static final char INDICATOR = 'f';
+
+  private AdditionalFieldsUtil() {
+  }
+
+  /**
+   * Adds field if it does not exist and a subfield with a value to that field
+   *
+   * @param record   record that needs to be updated
+   * @param field    field that should contain new subfield
+   * @param subfield new subfield to add
+   * @param value    value of the subfield to add
+   * @return true if succeeded, false otherwise
+   */
+  public static boolean addFieldToMarcRecord(Record record, String field, char subfield, String value) {
```

Comment:
Change to 'void' or use 'boolean' return value. Because for current implementation return value never using.